modules/network-observability-filtering-ebpf-rule.adoc

```diff
@@ -4,19 +4,22 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="network-observability-filtering-ebpf-rule_{context}"]
-= Filtering eBPF flow data using a global rule
-You can configure the `FlowCollector` to filter eBPF flows using a global rule to control the flow of packets cached in the eBPF flow table.
+= Filtering eBPF flow data using multiple rules
+You can configure the `FlowCollector` to filter eBPF flows using multiple rules to control the flow of packets cached in the eBPF flow table.
 
 .Procedure
 . In the web console, navigate to *Operators* -> *Installed Operators*.
 . Under the *Provided APIs* heading for *Network Observability*, select *Flow Collector*.
 . Select *cluster*, then select the *YAML* tab.
 . Configure the `FlowCollector` custom resource, similar to the following sample configurations:
-+
 
-[%collapsible]
-.Filter Kubernetes service traffic to a specific Pod IP endpoint
+[IMPORTANT]
+====
+You cannot use duplicate CIDRs in filter rules.
```

Comment:
There's another important remark that I think should be mentioned:

If an IP matches several rules, only the one with the longest CIDR prefix will be evaluated.